src/xenia/cpu/backend/x64/x64_seq_vector.cc

```diff
@@ -708,6 +708,29 @@ struct VECTOR_SHL_V128
 
   static void EmitInt8(X64Emitter& e, const EmitArgType& i) {
     // TODO(benvanik): native version (with shift magic).
+
+    if (i.src2.is_constant) {
+      unsigned char n = i.src2.constant().u8[0];
+      bool all = true;
+      for (unsigned j = 1; j < 16; ++j) {
+        if (i.src2.constant().u8[j] != n) {
+          all = false;
+
+          break;
+        }
+      }
+      if (all) {
+        vec128_t bleh;
+
+        uint8_t mask = ~(uint8_t)(((uint16_t)0x100) >> n);
+
+        for (unsigned i = 0; i < 16; ++i) bleh.u8[i] = mask;
```

Comment:
Put this in a proper code block. One liners like this are annoying to read and maintain.